contributors/guide/bug-bounty.md

```diff
@@ -74,6 +74,6 @@ vulnerability reports in these areas, they are not (currently) eligible to recei
 - Vulnerabilities specific to a hosted Kubernetes setup<br>
   _Please report these through the associated provider_
 
-[Product Security Team]: https://github.com/kubernetes/sig-release/blob/master/security-release-process-documentation/security-release-process.md#product-security-team-pst
+[Product Security Committee]: https://github.com/kubernetes/security/blob/master/security-release-process.md#product-security-committee-psc
```

Comment:
nit: can you use git.k8s.io links everywhere (just to be consistent across the repo)? :grimacing: 

This would become https://git.k8s.io/kubernetes/security/security-release-process.md#product-security-committee-psc.